### ISSUE TITLE
batch_size

### ISSUE THREAD

**Issue by liulxx** [aosheng1996/BUFFER#7]
Thanks for sharing your codes. When training the model, it seems that it does not support batch_size greater than 1. Will batch_size affect the accuracy of the model? Thank you.

**Comment by aosheng1996** [comment]
Hi, @liulxx , thanks for your interest in our work!

We use parts of the implementations of [D3Feat](https://github.com/XuyangBai/D3Feat.pytorch), so you can find the answer [here](https://github.com/XuyangBai/D3Feat.pytorch/issues/10#issuecomment-874189084). In our work, the batch_size doesn't affect the accuracy.

Best,
Sheng

**Comment by liulxx** [comment]
Thanks!
